lib/dns/proof.js

```diff
@@ -5,11 +5,29 @@ class Proof {
     this.sig = sig;
     this.sigwire = sigwire;
     this.rrdata = rrdata;
+    this.sigwiredata = Buffer.concat([sigwire, rrdata]);
+  }
+
+  toConcat(rrdata) {
+    let sigwiredatalength = this.sigwiredata.toString('hex').length / 2;
```

Comment:
It's not necessary to convert it to a string just to measure its length - just get its length directly (don't divide by two).